README.md

```diff
@@ -34,6 +39,28 @@ If you don't have the AsyncAPI Generator installed, you can install it like this
 npm install -g @asyncapi/generator
 ```
 
+### or 
+
+### Using AsyncAPI CLI
+
+Install template
+
+```bash
+npm install -g @asyncapi/html-template@0.16.0
+```
+
+Install AsyncAPI CLI
+
+```bash
+npm install -g @asyncapi/cli
+```
+
+Generate using CLI
+
+```bash
+asyncapi generate fromTemplate asyncapi.yaml @asyncapi/html-template
```

Comment:
if you copy paste this command, it will not work for you right? so maybe workaround it by having a link to `asyncapi.yaml` or add proper explanation

possible link to use could be https://raw.githubusercontent.com/asyncapi/spec/master/examples/streetlights-kafka.yml